opm/simulators/flow/TracerModel.hpp

```diff
@@ -843,14 +841,15 @@ class TracerModel : public GenericTracerModel<GetPropType<TypeTag, Properties::G
                     }
                 }
 
-                Scalar rateWellTotal = rateWellNeg + rateWellPos;
+                //Scalar rateWellTotal = rateWellNeg + rateWellPos;
```

Comment:
Might as well just remove this statement completely.

Reply:
ok